doc/release-notes-bucash.md

```diff
@@ -1,26 +1,27 @@
-Release Notes for Bitcoin Unlimited Cash Edition 1.5.0.2
+Release Notes for Bitcoin Unlimited Cash Edition 1.5.1.0-rc1
 =========================================================
 
-Bitcoin Unlimited Cash Edition version 1.5.0.2 is now available from:
+Bitcoin Unlimited Cash Edition release candidate 1 for version 1.5.1.0 is now available from:
 
   <https://bitcoinunlimited.info/download>
 
 Please report bugs using the issue tracker at github:
 
   <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
 
-This is a minor bugs fix only release version based of Bitcoin Unlimited compatible
+This is a main release candidate based of Bitcoin Unlimited compatible
```

Comment:
As this just caught my eye: Don't we want to get rid of 'based' here?

Also I wonder about formatting wrt. line-lengths. It looks to me this could profit from a reformat to <80 lines.

Reply:
fixed.

WRT 80 chars long line: the render of the markdown file are independent from the actual len of a line, but I think you have a point. 

I will start enforcing a max line length starting from the next release :smile: